tests/end_to_end/conftest.py

```diff
@@ -10,57 +9,10 @@ def login_for_end_to_end_testing(browser):
     # Open a new page and go to the staging site.
     context = browser.new_context()
     page = context.new_page()
-    page.goto(f"{E2E_TEST_URI}/")
```

Comment:
We can actually get rid of this method all together!

We only need the last two fixtures defined in this E2E config:

* `end_to_end_context`
* `authenticated_page`